packages/components/src/popover/types.ts

```diff
@@ -160,6 +160,12 @@ export type PopoverProps = {
 	 * @default false
 	 */
 	inline?: boolean;
+	/**
+	 * Inline styles to apply to the popover element.
+	 *
+	 * @default undefined
+	 */
+	style?: React.CSSProperties;
```

Comment:
Technically we don't have to list this type explicitly, because it's already included as a `div` prop via `WordPressComponentProps`, and excluded from the `MotionProps` omitting.

And I guess this comment can be a bit misleading:

> 		// To avoid overlaps between the standard HTML attributes and the props
> 		// expected by `framer-motion`, omit all framer motion props from popover
> 		// props (except for `animate`, `children`, and `style`, which are
> 		// re-defined in `PopoverProps`).

Maybe "except for `animate` and `children` which are re-defined in `PopoverProps`, and `style` which is merged safely"?

Reply:
Sounds good - done in c811da6. Thanks for the quick review!